llvm/lib/Target/AMDGPU/MIMGInstructions.td

```diff
@@ -1553,8 +1578,10 @@ defm IMAGE_ATOMIC_DEC           : MIMG_Atomic_Renamed <mimgopc<0x16, 0x16, 0x1c>
 defm IMAGE_ATOMIC_FCMPSWAP      : MIMG_Atomic <mimgopc<MIMG.NOP, MIMG.NOP, 0x1d, MIMG.NOP>, "image_atomic_fcmpswap", 1, 1>;
 defm IMAGE_ATOMIC_FMIN          : MIMG_Atomic <mimgopc<MIMG.NOP, MIMG.NOP, 0x1e, MIMG.NOP>, "image_atomic_fmin", 0, 1>;
 defm IMAGE_ATOMIC_FMAX          : MIMG_Atomic <mimgopc<MIMG.NOP, MIMG.NOP, 0x1f, MIMG.NOP>, "image_atomic_fmax", 0, 1>;
+defm IMAGE_ATOMIC_LOAD          : MIMG_NoSampler_NoDisasm <mimgopc<0x00, 0x00, 0x00>, "image_load", 1>;
```

Comment:
This is pretty hacky. "No disasm" is usually the realm of pseudo instructions, so can you make this a pseudo instruction?

Reply:
When making `IMAGE_ATOMIC_LOAD` a pseudo instr, it seems AsmParser is always parsing image_load as `IMAGE_ATOMIC_LOAD` which when made pseudo fails on MC layer. Is there a way to make AsmParser match `IMAGE_LOAD` first instead?